storage/innobase/handler/ha_innodb.cc

```diff
@@ -7919,7 +7949,7 @@ ha_innobase::build_template(
 	} else {
 no_icp:
 		/* No index condition pushdown */
-		m_prebuilt->idx_cond = NULL;
+		ut_ad(m_prebuilt->idx_cond == nullptr);
```

Comment:
Elsewhere in this function, we are checking for null pointers with the unary `!` operator. Doing so would also trim a few bytes off the assertion expression string in the debug executables.

Reply:
ok, sure.